File: CS/ScheduledTasksAPIClientDemo/ReportViewerForm.cs
using DevExpress.ReportServer.Printing;
using DevExpress.ReportServer.ServiceModel.DataContracts;
using System.Windows.Forms;

namespace ScheduledTasksAPIClientDemo {
    public partial class ReportViewerForm : Form {

        public ReportViewerForm(int reportId) {
            InitializeComponent();
            remoteDocumentSource1.ServiceUri = MainForm.ServerAddress;
            remoteDocumentSource1.AuthenticationType = AuthenticationType.Guest;
            remoteDocumentSource1.ReportIdentity = new GeneratedReportIdentity(reportId);
            remoteDocumentSource1.CreateDocument();
        }
    }
}

File: VB/ScheduledTasksAPIClientDemo/ReportViewerForm.vb
Imports DevExpress.ReportServer.Printing
Imports DevExpress.ReportServer.ServiceModel.DataContracts
Imports System.Windows.Forms

Namespace ScheduledTasksAPIClientDemo
    Partial Public Class ReportViewerForm
        Inherits Form

        Public Sub New(ByVal reportId As Integer)
            InitializeComponent()
            remoteDocumentSource1.ServiceUri = MainForm.ServerAddress
            remoteDocumentSource1.AuthenticationType = AuthenticationType.Guest
            remoteDocumentSource1.ReportIdentity = New GeneratedReportIdentity(reportId)
            remoteDocumentSource1.CreateDocument()
        End Sub
    End Class
End Namespace
